lib/presentation/home/pages/home_page.dart

```diff
@@ -36,10 +35,7 @@ class HomePageState extends State<HomePage> {
           ),
           StartTrainingButton(
             onPressed: () {
-              Navigator.push(
-                context,
-                MaterialPageRoute(builder: (context) => const TrainingStartPage()),
-              );
+              Navigator.pushNamed(context, '/training/start');
```

Comment:
die eigenschaft onPressed() gehört in die widget komponente und nicht hier her. bitte das noch ändern, dann werde ich's mergen